static/index.js

```diff
@@ -52,6 +52,7 @@ function appendOutput(content, className) {
 
 const editor = new CodeFlask("#editor-target", {
   language: "gleam",
+  defaultTheme: false
```

Comment:
We'll need syntax highlighting for sure

Reply:
I've added syntax highlighting and updated the screenshots in the PR description

I've based this entirely on the theme used in package docs sites, but I'm not sure if it works well here
- the comments are nearly illegible at this font size
- the darker background color for the code editor doesn't sit well with the design of the page
- also in case of the light theme the editor appears as disabled/readonly

Reply:
What if we don't use a distinct background colour for the editor?

Reply:
It would look like this

![localhost_8000_advanced-features_use-sugar_ (2)](https://github.com/gleam-lang/language-tour/assets/4096683/ec06527a-8a88-4659-bc31-9d90172dd0f9)

![localhost_8000_advanced-features_use-sugar_ (3)](https://github.com/gleam-lang/language-tour/assets/4096683/25fe89f5-e5aa-46ed-9e8d-ad11aff7d899)